core/src/main/scala/cats/Trivial.scala

```diff
@@ -1,23 +0,0 @@
-package cats
-
-/**
- * The "Unit type class".  The only instance of `Trivial` is given by
- * `Trivial.manifest`, and this instance is guaranteed to be in the
- * implicit scope.  Several convenience type aliases are provided in
- * companion object, covering a few common use cases and avoiding the
- * need for unnecessary lambdas (e.g. if you want a trivial type class
- * instance for a type constructor, you should use `Trivial.PH1`).
- */
-sealed trait Trivial
-
-object Trivial {
```

Comment:
I thought about making the same objection, but my conclusion was that "less is more" in terms of API surface area.  Shapeless already has a `Trivial`, and it is very likely to be on the classpath of anyone who needs this sort of thing.  I don't think there's a need for cats to keep it.